packages/react-components/react-card/etc/react-card.api.md

```diff
@@ -20,6 +20,7 @@ export const cardClassNames: SlotClassNames<CardSlots>;
 // @public
 export const cardCSSVars: {
     cardSizeVar: string;
+    cardBorderRadiusVar: string;
```

Comment:
Q: is this really intended to be public ?

Reply:
Yeah, it's best to export this and have consumers depend on something we control instead of trying to overwrite hardcoded strings and creating more problems. There are use cases where this is happening already.